api-client/src/index.ts

```diff
@@ -55,6 +55,7 @@ export {
   SHELTER_CONTACT_ENDPOINT_NAME,
 } from './endpoints/createShelterContactEndpoint'
 export { default as createCitiesEndpoint, CITIES_ENDPOINT_NAME } from './endpoints/createCitiesEndpoint'
+export { default as createCityEndpoint, CITY_ENDPOINT_NAME } from './endpoints/createCityEndpoint'
```

Comment:
Hmmm not that sure actually, just historically grown. Since all the things in the api-client should only be used via the exports from index.ts, it should not matter (so also changing it if there is a reason to change it would be okay).